lms/djangoapps/oauth2_handler/handlers.py

```diff
@@ -200,13 +200,13 @@ def _get_courses_with_access_type(self, user, access_type):
         course_ids = cache.get(key)
 
         if not course_ids:
-            courses = _get_all_courses()
+            courses = CourseOverview.get_all_course_keys()
 
             # Global staff have access to all courses. Filter courses for non-global staff.
             if not GlobalStaff().has_user(user):
                 courses = [course for course in courses if has_access(user, access_type, course)]
 
-            course_ids = [unicode(course.id) for course in courses]
+            course_ids = [unicode(course) for course in courses]
```

Comment:
Oh, nvm, didn't notice the change above. Could you rename `courses` to `course_keys` for clarity?